velox/type/TimestampConversion.cpp

```diff
@@ -144,6 +145,53 @@ bool isValidWeekDate(int32_t weekYear, int32_t weekOfYear, int32_t dayOfWeek) {
   return true;
 }
 
+bool isValidWeekOfMonthDate(
+    int32_t year,
+    int32_t month,
+    int32_t weekOfMonth,
+    int32_t dayOfWeek) {
```

Comment:
Would you add unit tests for this function? Please ensure all cases are covered in the test.

Reply:
I added [this tests](https://github.com/facebookincubator/velox/compare/main...NEUpanning:velox:daysSinceEpochFromWeekOfMonthDate#diff-140dc50e6dac735f72d29014da44b045509df0dd1737f458de1fe8cfd33d8145R158-R183) for invalid cases. Is this sufficient?

Reply:
I see. Thanks.